src/index.ts

```diff
@@ -138,6 +139,7 @@ export interface SpannerOptions extends GrpcClientOptions {
   sslCreds?: grpc.ChannelCredentials;
   routeToLeaderEnabled?: boolean;
   directedReadOptions?: google.spanner.v1.IDirectedReadOptions | null;
+  observabilityConfig?: ObservabilityOptions | undefined;
```

Comment:
We don't need to use "undefined" if a field is marked optional using "?"

Also please add the documentation for the new options similar to other options. 

We will also need to define "Observability Options" type

Reply:
What's the precedent on line 141 `directedReadOptions?: google.spanner.v1.IDirectedReadOptions | null;`?

ObservabilityOptions was defined in src/instrument.ts

Reply:
I've exposed ObservabilityOptions as a type itself.